doc/source/api/plotting/index.rst

```diff
@@ -35,7 +35,17 @@ all plotting functionality in PyVista.
    CameraPosition
    CornerAnnotation
    CubeAxesActor
+   CustomInteractorStyle
```

Comment:
This doesn't work presumably because these aren't in the `pyvista` namespace.  I'm thinking we should use a separate heading here like `Interactor Style API` to include these.